geospaas_processing/cli/download.py

```diff
@@ -0,0 +1,87 @@
+"""
+Script for cli of downloading repo. This script accepts the argunemt from sys.args by the
+help of python "argparse" and download the dataset files based on the input arguments.
+"""
+import argparse
+import os
+import subprocess
+import sys
+import time
+from datetime import datetime
+from distutils.util import strtobool
+import json
+import django
+import ast
+
+from dateutil.parser import parse
+from dateutil.relativedelta import relativedelta
+from dateutil.tz import tzutc
+from django.contrib.gis.geos import GEOSGeometry
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'geospaas_processing.settings')
+django.setup()
+from geospaas.catalog.models import Dataset
+import geospaas_processing.downloaders as downloaders
+
+
+def main(arg):
+    cumulative_query = {}
+    if arg.query:
+        query_list = json.loads(arg.query)
+        for query in query_list:
+            cumulative_query.update(query)
+    if arg.rel_time_flag:
+        designated_begin = datetime.now().replace(tzinfo=tzutc()) + relativedelta(
+            hours=-abs(int(arg.begin)))
+        designated_end = datetime.now().replace(tzinfo=tzutc())
+    else:
+        designated_begin = datetime.strptime(arg.begin, "%Y-%m-%d").replace(tzinfo=tzutc())
+        designated_end = datetime.strptime(arg.end, "%Y-%m-%d").replace(tzinfo=tzutc())
+    download_manager = downloaders.DownloadManager(
+        download_directory=arg.down_dir.rstrip(os.path.sep),
+        provider_settings_path=arg.config_file,
+        max_downloads=int(arg.number_per_day)*(((designated_end-designated_begin).days)+1),
+        use_file_prefix=arg.use_filename_prefix,
+        time_coverage_start__gte=designated_begin,
+        time_coverage_end__lte=designated_end,
+        geographic_location__geometry__intersects=GEOSGeometry(arg.geometry),
+        **cumulative_query
+    )
+    download_manager.download()
+
+def argparse_define():
```

Comment:
Try to use verb as the first name in the function and don't forget dockstrings.
```suggestion
def parse_args():
""" Parse the script input arguments """ 
```